src/Symfony/Component/HttpClient/Tests/MockHttpClientTest.php

```diff
@@ -167,7 +210,7 @@ protected function getHttpClient(string $testCase): HttpClientInterface
             case 'testResolve':
                 $responses[] = new MockResponse($body, ['response_headers' => $headers]);
                 $responses[] = new MockResponse($body, ['response_headers' => $headers]);
-                $responses[] = new MockResponse((function () { throw new \Exception('Fake connection timeout'); yield ''; })(), ['response_headers' => $headers]);
+                $responses[] = new MockResponse((function () { yield ''; })(), ['response_headers' => $headers]);
```

Comment:
We don't need to throw for the test to be effective. Yielding after throwing "breaks" PHP 7.2.